modules/tinymce/src/core/main/ts/content/SetContentImpl.ts

```diff
@@ -35,9 +35,11 @@ const moveSelection = (editor: Editor): void => {
   }
 };
 
-const setEditorHtml = (editor: Editor, html: string): void => {
+const setEditorHtml = (editor: Editor, html: string, noSelection: boolean = false): void => {
```

Comment:
This isn't really accurate, as we're always passing a value here. So can we please update the types to reflect that?
```suggestion
const setEditorHtml = (editor: Editor, html: string, noSelection: boolean | undefined): void => {
```